preupg/settings.py

```diff
@@ -43,6 +43,15 @@
 cache_dir = "/var/cache/preupgrade"
 log_dir = "/var/log/preupgrade"
 
+# file with meta info
+properties_ini = "properties.ini"
+# section inside ini file
+section_name = "preupgrade-assistant-modules"
+# key which specify source major OS version
+src_version_key ="srcMajorVersion"
+# key to specify destination major OS version
+dest_version_key ="destMajorVersion"
```

Comment:
I have no problem with @AloisMahdal approach. The probability that somebody would create a directory for something else than modules in the default place for modules (/usr/share/preupg/) and put properties.ini with some arbitrary content in it is so close to zero, that we can ignore it.